_config.yml

```
@@ -64,4 +64,13 @@ plugins: []
   #- jekyll-feed
 
 feed:
-  title: Dripline
+  title: "Hypha Worker Co-operative"
+  collections:
+    posts:
+      path: "feed.xml"
+      name: "Dripline"
+      description: "This is the place for our collective writing ranging from short posts, to interviews, design retrospectives, and essays. We publish on a (mostly) monthly schedule."
```

Not blocking but since we're cleaning stuff up 😄  I think this would sound better with fewer words:

```suggestion
      description: "Our collective writing ranging from short posts, to interviews, design retrospectives, and essays. Published on a (mostly) monthly schedule."
```